ReactAndroid/src/main/java/com/facebook/react/uimanager/MatrixMathHelper.java

```diff
@@ -350,4 +381,79 @@ public static void quaternionToDegreesXYZ(double[] q, double[] result) {
   public static double roundTo3Places(double n) {
     return Math.round(n * 1000d) * 0.001;
   }
+
+  public static double[] createIdentityMatrix() {
+    double[] res = new double[16];
+    resetIdentityMatrix(res);
+    return res;
+  }
+
+  public static double degreesToRadians(double degrees) {
+    return degrees * Math.PI / 180;
+  }
+
+  public static void resetIdentityMatrix(double[] matrix) {
+    matrix[1] = matrix[2] = matrix[3] = matrix[4] = matrix[6] = matrix[7] = matrix[8] = matrix[9] =
+      matrix[11] = matrix[12] = matrix[13] = matrix[14] = 0;
+    matrix[0] = matrix[5] = matrix[10] = matrix[15] = 1;
+  }
+
+  public static void applyPerspective(double[] m, double perspective) {
+    m[11] = -1 / perspective;
```

Comment:
Sounds good. Sometimes the value will unintentionally pass over 0 during an animation, but I agree this is probably not the time or the place to handle for that. Just thought I would mention it. Thanks :)